File: commands/rules.py
rules = ['Be kind, civil and respectful to one another.', 'Ask before self-promoting.', 'Do not spam messages in the chats.', 'List to the moderators.', 'Don\'t argue with each other.', 'Have fun!']

async def execute(message, vars):
    resp = 'The rules and guidelines for the stream and this server are:'
    maxlen = len(str(len(rules)+1))
    for idx, val in enumerate(rules):
        resp += f'\n\t`{str(idx+1).rjust(maxlen)}.` {val}'
    return await message.channel.send(resp)

cmd = {
	'command': 'rules',
	'aliases': ['rule', 'guidelines'],
	'description': 'displays the rules for the server.',
	'run': execute
}

File: discord-bot-djs/commands/rules.js
const { SlashCommandBuilder } = require('@discordjs/builders');
const { MessageEmbed } = require('discord.js');

module.exports = {
	name: 'rules',
	comm: new SlashCommandBuilder().setName('rules').setDescription('Display the rules of the server'),
	exec: async (interaction, client, main) => {

		const rules = [
			'Be kind, civil and respectful to one another.',
			'Ask before self-promoting.',
			'Do not spam messages in the chats.',
			'Listen to the moderators.',
			'Don\'t argue with each other.',
			'https://cdn.discordapp.com/attachments/336935738581188609/782997977173196840/unknown.png',
			'Have fun'
		]

		await interaction.reply({
				embeds: [
					new MessageEmbed(main.exports.embed).setTitle('Rules:').setDescription(
						rules.map((rule, index) => `${index + 1}. ${rule}`).join`\n`
					)
				]
			});

	}
}

